corio.py

```diff
@@ -303,7 +310,7 @@ def main(options):
             activate_degraded_mode(options)
             options.number_of_nodes -= literal_eval(os.getenv("DEGRADED_PODS_CNT"))
         start_processes(processes)
-        while True:
+        while processes:
```

Comment:
how the processes will be triggered one after another ?

Reply:
This will run till process dict is not empty.